src/test/java/org/takes/rq/RqMultipartTest.java

```diff
@@ -494,22 +545,30 @@ public void producesPartsWithContentLength() throws IOException {
                 ),
                 Joiner.on("\r\n").join(
                     "--AaB01x",
-                    "Content-Disposition: form-data; name=\"t2\"",
+                    new StringBuilder()
```

Comment:
@raphaelln please read http://www.yegor256.com/2014/06/19/avoid-string-concatenation.html - `StringBuilder` is not a proper replacement